tests/wrappers/core_wrappers_test.py

```diff
@@ -264,11 +272,9 @@ def reward(self, reward):
     env.reset()
     _, reward, _, _ = env.step(0)
     assert reward == -5
-    assert env.episode_reward == -5
```

Comment:
I think these two deleted lines can be restored now that you're overloading `multistep()` rather than `raw_step()`

Reply:
Done.